tests/10-api.spec.js

```diff
@@ -80,6 +81,30 @@ app.get('/test/:id',
     res.json({message: 'Get request was successful.'});
   });
 
+app.post('/revoke',
+  authorizeZcapRevocation({
+    documentLoader,
+    suiteFactory() {
+      return new Ed25519Signature2020();
+    },
+    getRootController() {
+      // root controller(Admin DID)
+      return 'did:key:z6Mkfeco2NSEPeFV3DkjNSabaCza1EoS3CmqLb1eJ5BriiaR';
+    },
+    getExpectedTarget() {
+      return {
+        expectedTarget: `${BASE_URL}/revoke`
```

Comment:
I don't recommend we do this at all. Rather, we should use the `/revocations/` subpath as implemented in ezcap-express so that others copy that pattern instead. So, the client should hit `<base url>/revocations/<id of zcap to revoke>`. We can have two tests, one that hits that and another that hits `<base url>/revocations` to exercise both code paths.

Reply:
@dlongley I've updated the tests to hit `<base url>/revocations/:id` here 42ce811.  I'm not clear about the second test which will be hitting `<base url>/revocations`. What exactly are we testing?

Reply:
This code path: https://github.com/digitalbazaar/ezcap-express/pull/18/files#diff-c2f4bce4f8670313ae1e7542403369157e228062ae9f33da87bba9473a0d2f04L140

If `/revocations/` (with the slash) is not in the request URL we'll hit that conditional. Otherwise, we'll fall through -- so we want to hit both paths.

Reply:
Added here 2867cda